README.md

```diff
@@ -1,8 +1,10 @@
 # Google Student Training in Engineering Program
 
-This repo contains the projects you'll work on for the first half of STEP.
+This repo contains the projects you'll will work on for the first half of STEP.
 This work is based on the [Google Software Product Sprint](https://g.co/softwareproductsprint) program.
 
+testing
+
```

Comment:
I've applied the changes

Reply:
For this one I was referring to the "will" and the "testing", which don't fit.  It would be better to personalize the README a bit (i.e. say that it's your portfolio)

Reply:
Whoops, needed to refresh sorry

Reply:
okay I think I correctly added the files in the right places now

Reply:
It seems further now, I'm seeing over 200 deleted files